libraries/python/cfengine.py

```diff
@@ -44,6 +44,13 @@ def _would_log(level_set, msg_level):
         # uknown level, assume it would be logged
         return True
 
+    # info: log messages are special because they report changes done in promise
+    # evaluation which is important not only for showing to the user, but also
+    # for auditing/changelog and all modules are required to send info: messages
+    # for all REPAIRED promises.
+    if msg_level == "info":
```

Comment:
@vpodzime so it needs to be something like:
```suggestion
    if msg_level in ("info", "error", "notice", "critical", "warning"):
```

Also, `_would_log` doesn't fit this. I think you should either move this check to the `_log` function, or change the name of this function from `_would_log` to `_should_send_log_message` or something similar.